packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -702,7 +675,7 @@ function escapeSecretPlaceholders(variables: Record<string, string>) {
 
 // backup existing ARM template and parameter files to backup folder named with current timestamp
 async function backupExistingFilesIfNecessary(ctx: SolutionContext): Promise<void> {
-  const armBaseFolder = path.join(ctx.root, baseFolder);
+  const armBaseFolder = path.join(ctx.root, configsFolder);
```

Comment:
will the backup logic be updated later in this PR?

Reply:
since this PR merge in dev for daily release, will support backup logic for another PR.